src/nci_adapter.c

```diff
@@ -430,7 +588,9 @@ nci_adapter_nci_intf_activated(
             tag = nci_adapter_create_known_tag(self, ntf);
         }
         if (!tag) {
-            nfc_adapter_add_other_tag(NFC_ADAPTER(self), self->target);
+            NfcParamPoll poll;
```

Comment:
Empty line after the declaration pls! That's the last one, I promise))

Reply:
thanks, I have fixed :+1: